src/site/content/es/authors/index.njk

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I think having these .njk files is going to be an issue because for every language we will need to duplicate this file, and if we make changes to the en version, we have to remember to change it in every language.

On d.c.c. we have all of our .njk files in the _includes directory and site/content only contains markdown files. Could we do that here for these files?

Reply:
Ok, I just broke them into layouts, which works just fine for this!